src/Crisp.jsx

```diff
@@ -36,7 +36,7 @@ class Crisp extends React.Component {
     if (crispTokenId !== '') {
       window.CRISP_TOKEN_ID = crispTokenId;
     }
-    if (crispRuntimeConfig.length > 0) {
+    if (crispRuntimeConfig !== {}) {
```

Comment:
maybe `Object.keys(crispRuntimeConfig).length > 0` ?

Reply:
You can't use equality with JS object as it will check if this is the same object and not if these two objects have the same content.

```
# ({} !== {})
true
```